src/init.cpp

```diff
@@ -410,6 +410,7 @@ std::string HelpMessage(HelpMessageMode mode)
     strUsage += HelpMessageOpt("-salvagewallet", _("Attempt to recover private keys from a corrupt wallet.dat") + " " + _("on startup"));
     strUsage += HelpMessageOpt("-sendfreetransactions", strprintf(_("Send transactions as zero-fee transactions if possible (default: %u)"), 0));
     strUsage += HelpMessageOpt("-spendzeroconfchange", strprintf(_("Spend unconfirmed change when sending transactions (default: %u)"), 1));
+    strUsage += HelpMessageOpt("-txblockexpiry", strprintf(_("Set the number of blocks after which an unconfirmed transaction expires from the mempool (default: %u)"), DEFAULT_TX_EXPIRY_DELTA));
```

Comment:
The current message sounds more like a config option for #2384 than transaction expiry. Maybe something like `Set the number of blocks after which a created transaction will become invalid if it has not been mined`?

@lindanlee thoughts on user messaging?

Reply:
`Set the maximum number of blocks within which this transactions must either confirmed or resent`

Reply:
`Set the maximum number of blocks within which transactions must be resent if they have not been confirmed` ?